db/fixtures/chargeback_rates.yml

```diff
@@ -198,18 +185,6 @@
           :finish: Infinity
           :fixed_rate: 1.0
           :variable_rate: 0.02
-    - :description: Allocated CPU Count
-      :group: cpu
-      :source: allocated
-      :metric: derived_vm_numvcpus
-      :per_time: hourly
-      :per_unit: cpu
-      :type_currency: Dollars
-      :tiers:
-        - :start: 0
-          :finish: Infinity
-          :fixed_rate: 1.0
-          :variable_rate: 0.0
```

Comment:
Needs clarification: Do containers collect `derived_vm_numvcpus`?

If not -> we should drop `Used CPU Cores` as well. (Lookup the method calculating `Used CPU Cores` -- `v_derived_cpu_total_cores_used`)

if yes -> why to drop the possibility when we collect it?